test/unit/src/server/handlers/t_login.js

```diff
@@ -63,15 +65,11 @@ describe('processVerifiedAssertion', () => {
     let processVerifiedAssertion;
 
     beforeEach(() => {
-      processVerifiedAssertion = proxyquire(
-        '../../../../../src/server/handlers/login',
-        {
-          '../helpers/config': {
-            conf: Map({ UBUNTU_SSO_URL, OPENID_VERIFY_URL, GITHUB_API_ENDPOINT }),
-            '@noCallThru': true
-          }
-        }
-      ).processVerifiedAssertion;
+      ({ processVerifiedAssertion } = requireWithMockConfig({
```

Comment:
Why the parents `()` around whole thing?

I also see a bit different pattern here and in test above: `processVerifiedAssertion = ...processVerifiedAssertion` vs `{ processVerifiedAssertion } = ...`. Would be good to use one of them consistently.